grails-app/conf/default-conservation-lists.json

```diff
@@ -0,0 +1,5 @@
+{
+  "defaultSourceField": "status",
```

Comment:
Where do the real list config files live? I see defaults and examples here - are they in ala-install?